### DISCUSSION TITLE
What do cgroups actually limit?

### DISCUSSION THREAD

**Discussion by SgtCoin** [filecoin-project/lotus#7972] [General Q&A]
TSE note: 

`What does cgroups actually limit? Is it just the worker process or other processes as well? `

Question from community: 

`re. v1.13.2's "Workers are now aware of cgroup memory limits"  Does anyone know if it also applies to the internal worker processes "spawned" by lotus-miner if the miner is configured to do PC1, too?   or does the new feature apply to just the "lotus-worker" program?`

**Comment by SgtCoin** [comment]
I am going to test cgroups on devnet and report the findings.

**Comment by LexLuthr** [comment]
@SgtCoin Can we close this out or maybe delete it?

https://lotus.filecoin.io/storage-providers/seal-workers/seal-workers/#control-groups
